src/components/Dashboard/SafeApps/Grid.tsx

```diff
@@ -42,11 +40,16 @@ const StyledIcon = styled.img`
   height: auto;
 `
 
+const MAX_APPS = 3
+
+const OFFICIAL_APPS: Record<string, string> = {
+  TRANSACTION_BUILDER: '29',
+  WALLET_CONNECT: '11',
```

Comment:
No need for a hash map here. An array like before is sufficient. We can add comments with the app names.

Reply:
Will put it back together with the comments we had

Reply:
Why? If we needed to change the IDs, it is not so clear.

Reply:
We plan to have "official" apps flagged on the remote-config, so this isn't permanent anyway.